### ISSUE TITLE
what's the diffrence between "all_masks" and "all_masks_valid" in dataloader

### ISSUE THREAD

**Issue by YeJiangnan1029** [bennyguo/nerfren#7]
Hello! I'm trying to apply your great work in my self-construct systhetic dataset，so I attempt to complete the code for blender-format dataset according to the "rffr_dataset.py". However, I am confused about the meaning of "all_masks" and "all_masks_valid" to the training settting.
It seems that the "all_masks" defines the area of glass in a certain picture and only when the "all_masks_valid" is 1 it enables the "all_masks". I wonder if I'm correct, thannnnnnnnnnnks!

**Comment by bennyguo** [comment]
Hi! You're right. In training, we only use masks with all_masks_valid=1, and all_masks_valid is determined by
- whether an image is annotated with mask (=0 if not annotated)
- the number of masks to use (opt.num_masks)

**Comment by YeJiangnan1029** [comment]
i see, thank you!
